frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/user-templates.tsx

```diff
@@ -33,9 +34,18 @@ export const UserTemplates: React.FC<UserTemplatesProps> = React.memo(
       >
         <FormField isDisabled={!hasUserTemplates}>
           <FormSelect onChange={nullOnEmptyChange(onChange, VMSettingsField.USER_TEMPLATE)}>
-            <FormSelectPlaceholderOption
-              placeholder={hasUserTemplates ? NO_TEMPLATE : NO_TEMPLATE_AVAILABLE}
-            />
+            {hasUserTemplates && (
+              <FormSelectPlaceholderOption
+                placeholder={SELECT_TEMPLATE}
+                isDisabled={!hasFieldValue}
+              />
+            )}
+            {hasUserTemplates && (
+              <FormSelectOption key={NO_TEMPLATE} value="" label={NO_TEMPLATE} />
+            )}
+            {!hasUserTemplates && (
+              <FormSelectPlaceholderOption placeholder={NO_TEMPLATE_AVAILABLE} />
```

Comment:
it would be probably better to add keys to other render options as well

Reply:
good point, added